website/content/docs/k8s/installation/install.mdx

```diff
@@ -17,20 +17,69 @@ a server running inside or outside of Kubernetes.
 
 You can install Consul on Kubernetes using the following methods:
 
-1. [Helm chart install](#helm-chart-installation)
 1. [Consul K8s CLI install <sup>BETA</sup>.](#consul-k8s-cli-installation)
+1. [Helm chart install](#helm-chart-installation)
 
 Refer to the [architecture](/docs/k8s/installation/install#architecture) section to learn more about the general architecture of Consul on Kubernetes.
 For a hands-on experience with Consul as a service mesh
 for Kubernetes, follow the [Getting Started with Consul service
 mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh-deploy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
 
+## Consul K8s CLI Installation
+
+The recommended way to install Consul on Kubernetes for single cluster use cases is via the [Consul K8S CLI](/docs/k8s/k8s-cli). 
+You can install Consul on Kubernetes using the Consul K8s CLI tool after installing the CLI. 
+Ensure that `kubectl` is already configured to authenticate to the Kubernetes cluster using a valid `kubeconfig` file. 
+The Consul K8s CLI is currently available as an beta release and is not recommended for production environments.
```

Comment:
```suggestion
We recommend using the [Consul K8S CLI](/docs/k8s/k8s-cli) to install Consul on Kubernetes for single-cluster deployments. You can install Consul on Kubernetes using the Consul K8s CLI tool after installing the CLI. 

-> **Consul K8s CLI is currently in beta**: The tools is not recommended for production environments.

Before beginning the installation process, verify that `kubectl` is already configured to authenticate to the Kubernetes cluster using a valid `kubeconfig` file. 

The [Homebrew](https://brew.sh) package manager is required to complete the following installation instructions.
```
Just trying to make the language a little cleaner; also mentioned that the CLI tool installation procedure uses brew.